first_and_last.py

```diff
@@ -0,0 +1,50 @@
+# Time Complexity : log n
+# Space Complexity : n
+# Did this code successfully run on Leetcode : Yes
+# Any problem you faced while coding this : No major problems. Yesterday's progblems helped me form an 
```

Comment:
Good, the time you take to problems should slowly go down as the intuition starts to build up.